api/src/test/java/io/opencensus/common/TimestampTest.java

```diff
@@ -135,7 +135,19 @@ public void timestampSubtractTimestamp_NegativeResult() {
   }
 
   @Test
-  public void testTimestampEqual() {
+  public void timestamp_CompareTo() {
+    assertThat(Timestamp.create(0, 0).compareTo(Timestamp.create(0, 0))).isEqualTo(0);
+    assertThat(Timestamp.create(24, 42).compareTo(Timestamp.create(24, 42))).isEqualTo(0);
+    assertThat(Timestamp.create(-24, 42).compareTo(Timestamp.create(-24, 42))).isEqualTo(0);
```

Comment:
Could you please add a test for negative time without equality?
e.g. T(-24,1) happens before T(-24,2) ?